editor/import/scene_import_settings.cpp

```diff
@@ -377,9 +377,10 @@ void SceneImportSettings::_update_view_gizmos() {
 			continue;
 		}
 
-		MeshInstance3D *collider_view = static_cast<MeshInstance3D *>(mesh_node->find_node("collider_view"));
-		CRASH_COND_MSG(collider_view == nullptr, "This is unreachable, since the collider view is always created even when the collision is not used! If this is triggered there is a bug on the function `_fill_scene`.");
+		TypedArray<Node> descendants = mesh_node->find_nodes("collider_view", "MeshInstance3D");
+		CRASH_COND_MSG(descendants.is_empty(), "This is unreachable, since the collider view is always created even when the collision is not used! If this is triggered there is a bug on the function `_fill_scene`.");
 
+		MeshInstance3D *collider_view = static_cast<MeshInstance3D *>(descendants[0].operator Node *());
```

Comment:
`operator Node *()` was removed in #57205, but you can use `operator Object *()` which should work fine too.